client/pagination.go

```diff
@@ -0,0 +1,67 @@
+package client
+
+import "strconv"
+
+const limit = 100
+
+type Pagination struct {
+	offset int
+	params map[string]string
+}
+
+// Note: all the FIXME should be uncommented when Alpine adds support for go 1.18.
```

Comment:
Harnes uses terraform docker.
Alpine 3.15 (which terraform docker uses) supports golang 3.17.
Will have to wait for Alpine 3.16 which runs golang 3.18.

Once this happens we can switch back to generics (which is a new feature in golang 3.18).

Note: the env0-terraform library builds via go1.18, the issue is strictly related to integration tests (Harnes).